projects/igniteui-angular/src/lib/badge/badge.component.ts

```diff
@@ -60,7 +59,7 @@ export class IgxBadgeComponent {
     * ```
     */
     @Input()
-    public type: string | Type = 'default';
+    public type: string | IgxBadgeType = '';
```

Comment:
Just noticed the API doc on top (can't comment there) - but it does say "Allowed values are `default`...."

Reply:
yep, updated.